packages/vuetify/test/unit/components/VTimePicker/VTimePickerTitle.spec.js

```diff
@@ -2,126 +2,147 @@ import VTimePickerTitle from '@/components/VTimePicker/VTimePickerTitle'
 import { test } from '@/test'
 
 test('VTimePickerTitle.js', ({ mount }) => {
-  it('should render component in 24hr', () => {
-    const wrapper = mount(VTimePickerTitle, {
-      propsData: {
-        hour: 14,
-        minute: 13,
-        period: 'pm',
-        ampm: false
-      }
+  [true, false].forEach(useSecondsValue => {
+    const useSecondsDesc = (useSecondsValue ? '. with useSeconds' : '')
+    it('should render component in 24hr' + useSecondsDesc, () => {
+      const wrapper = mount(VTimePickerTitle, {
+        propsData: {
+          hour: 14,
+          minute: 13,
+          second: 25,
+          period: 'pm',
+          ampm: false,
+          useSeconds: useSecondsValue
+        }
+      })
+
+      expect(wrapper.html()).toMatchSnapshot()
     })
 
-    expect(wrapper.html()).toMatchSnapshot()
-  })
-
-  it('should render component in 12hr', () => {
-    const wrapper = mount(VTimePickerTitle, {
-      propsData: {
-        hour: 14,
-        minute: 13,
-        period: 'pm',
-        ampm: true
-      }
+    it('should render component in 12hr' + useSecondsDesc, () => {
+      const wrapper = mount(VTimePickerTitle, {
+        propsData: {
+          hour: 14,
+          minute: 13,
+          second: 25,
+          period: 'pm',
+          ampm: true,
+          useSeconds: useSecondsValue
+        }
+      })
+
+      expect(wrapper.html()).toMatchSnapshot()
     })
 
-    expect(wrapper.html()).toMatchSnapshot()
-  })
-
-  it('should render component when selecting hour', () => {
-    const wrapper = mount(VTimePickerTitle, {
-      propsData: {
-        hour: 14,
-        minute: 13,
-        period: 'pm',
-        selectingHour: true
-      }
+    it('should render component when selecting hour' + useSecondsDesc, () => {
+      const wrapper = mount(VTimePickerTitle, {
+        propsData: {
+          hour: 14,
+          minute: 13,
+          second: 25,
+          period: 'pm',
+          selecting: 1,
```

Comment:
export constants from the component and use them here instead of 1/2/3

Reply:
ready